packages/taco/src/dkg.ts

```diff
@@ -161,9 +164,7 @@ export class DkgClient {
   ): Promise<DkgRitual> {
     const ritual = await DkgClient.getRitual(provider, domain, ritualId);
     if (ritual.state !== DkgRitualState.FINALIZED) {
-      throw new Error(
-        `Ritual ${ritualId} is not finalized. State: ${ritual.state}`,
-      );
+      throw new Error(ERR_RITUAL_FAILED(ritualId, ritual));
```

Comment:
If the ritual state is not finalized, it could have failed, but it also could have not yet completed...or does something else before this call ensure that it is completed?
```suggestion
      throw new Error(ERR_RITUAL_NOT_FINALIZED(ritualId, ritual));
```